### ISSUE TITLE
Ability to upload to Nextcloud

### ISSUE THREAD

**Issue by d-Rickyy-b** [d-Rickyy-b/backmeup#21]

**Comment by tyutyutyu** [comment]
If you want to keep this tool lightweight, don't implement any upload feature (eg. cloud upload, ftp, scp). There are other great tools for that job (like [rclone](https://rclone.org/)).
It was just a thought.

**Comment by d-Rickyy-b** [comment]
Yeah, you are right. This was more an idea from another user. I created the issue to keep it in mind. Thanks for your opinion.
